Pipfile.lock

```diff
@@ -0,0 +1,57 @@
+{
+    "_meta": {
+        "hash": {
+            "sha256": "52b3f24c784d7b7a3638d90858fd8bff46f49ac8423d2c0a37ff5f97c94f9028"
+        },
+        "pipfile-spec": 6,
+        "requires": {
+            "python_version": "3.6"
+        },
+        "sources": [
+            {
+                "name": "pypi",
+                "url": "https://pypi.org/simple",
+                "verify_ssl": true
+            }
+        ]
+    },
+    "default": {
+        "lxml": {
+            "hashes": [
```

Comment:
from [an official example](https://github.com/pypa/pipfile#pipfilelock) this high number of hashes seems normal